test/test_map.jl

```diff
@@ -79,6 +79,7 @@ import Thermodynamics.Parameters.ThermodynamicsParameters
         :isobaric_specific_heat_liquid => :cp_l,
         :latent_heat_vaporization_at_reference => :LH_v0,
         :temperature_saturation_adjustment_min => :T_min,
+        # :temperature_saturation_adjustment_init_min => :T_init_min, # will need updated
```

Comment:
Should we delete that one?

Reply:
I figured I'd leave it for when we (eventually) update the compat. We can't right now, because Thermodynamics doesn't yet use this variable. Having TD itself depend on CP and the CP test env depend on TD makes updating dependencies a bit awkward.